doc/classes/Engine.xml

```diff
@@ -337,6 +337,10 @@
 			[b]Note:[/b] This property does not impact the editor's Errors tab when running a project from the editor.
 			[b]Warning:[/b] If set to [code]false[/code] anywhere in the project, important error messages may be hidden even if they are emitted from other scripts. In a [code]@tool[/code] script, this will also impact the editor itself. Do [i]not[/i] report bugs before ensuring error messages are enabled (as they are by default).
 		</member>
+		<member name="print_to_stdout" type="bool" setter="set_print_to_stdout" getter="is_printing_to_stdout" default="true">
+			If [code]false[/code], stops printing messages to the console and editor Output log. This property is equivalent to the [member ProjectSettings.application/run/disable_stdout] project setting.
+			[b]Note:[/b] This stops printing any errors or warnings produced by scripts to the Output log, but not to the console, for more details see [member print_error_messages].
```

Comment:
Any suggestions for more details and wording here is welcome

Reply:
The note seems to directly contradict the line just above it:

> stops printing messages to the console and editor Output log.

> This stops printing any errors or warnings produced by scripts to the Output log, **but not to the console,**

From my testing, stdout is disabled in all these 3 areas:

- Console
- Editor output panel
- File logging (as it's a byproduct of console output)

Reply:
Will take a look, I think the meaning here is that messages, with `print` are blocked, but errors are still printed to the console, will confirm

Reply:
Should be clearer now, also added a cross-ref to `print`